### PR TITLE
show only releases from followed artists

### PR DESCRIPTION
**Pull request by DanielW1987**
It's now possible to show releases only from your followed artists.

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Releases page: show only relases of followed artists](https://trello.com/c/5d0wAcxJ/229-releases-page-show-only-relases-of-followed-artists)

File: webapp/src/main/resources/static/ts/src/service/releases-render-service.ts [hunk 1]
```diff
@@ -25,17 +25,29 @@ export class ReleasesRenderService extends AbstractRenderService<ReleasesRespons
     }
 
     protected onRendering(data: ReleasesResponse): void {
-        data.items.forEach(release => {
-            const releaseDivElement = this.renderReleaseCard(release);
-            this.hostElement.insertAdjacentElement("beforeend", releaseDivElement);
-        });
+        console.log(data.items);
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/service/releases-render-service.ts:28]: Is the logging on purpose still here?

File: webapp/src/main/resources/static/ts/src/service/releases-service.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    constructor(releasesRestClient: ReleasesRestClient, releasesRenderService: ReleasesRenderService, urlService: UrlService) {
+        this.releasesRestClient = releasesRestClient;
+        this.releasesRenderService = releasesRenderService;
+        this.urlService = urlService;
+        this.getDocumentElements();
+        this.initFilterValuesFromUrl();
+        this.addReleaseFilterEventListener();
+    }
+
+    private getDocumentElements(): void {
```
Comment (hedgehog1833) [inline webapp/src/main/resources/static/ts/src/service/releases-service.ts:27]: Shouldn't this be `setDocumentElements()`? A void get-Method sound weird. And yes you are getting the elemnts but also setting them..

**Review by hedgehog1833 — COMMENTED** [review]
Looks nice!

**Review by hedgehog1833 — APPROVED** [review]

**Pull request merged**